kubeflow/trainer/rhai/transformers_instrumentation.py

```
@@ -0,0 +1,1450 @@
+                                if hasattr(state, "epoch") and state.epoch
+                                else 0.0
+                            ),
+                            "totalEpochs": total_epochs,
+                            "trainMetrics": dict(_progression_metrics_state.trainMetrics),
+                            "evalMetrics": dict(_progression_metrics_state.evalMetrics),
+                        }
+
+                        with open("/dev/termination-log", "w") as f:
+                            json.dump(termination_message, f)
+                    print("[Kubeflow] Final metrics written to termination message", flush=True)
+                except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)